reference/docs/conversion/sakai_10_6-10_7_oracle_conversion.sql

```diff
@@ -0,0 +1,3 @@
+-- SAK-29442 Sequence LB_PEER_EVAL_RESULT_S Missing
+CREATE SEQUENCE "LB_PEER_EVAL_RESULT_S" MINVALUE 1 MAXVALUE 999999999999999999999999999 INCREMENT BY 1 START WITH 1 CACHE 20 NOORDER NOCYCLE;
```

Comment:
It seems like this statement should be more minimal. Are the CACHE and NOORDER NOCYCLE necessary?